src/css/style.css

```diff
@@ -93,6 +93,14 @@ a {
 
 }
 
+@media only screen and (min-width: 768px) {
+
+    /* make dropdown menu's hoverable on desktop */
+    .dropdown:hover .dropdown-menu {
```

Comment:
This works, to also give the parent the 'active' background + let this work on a small window width, I personally think it is better to mimic the standard behaviour, i.e. adding the class 'open' to .dropdown.